src/EditorFeatures/Core/GlobalUsings.cs

```diff
@@ -0,0 +1,5 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+global using EditorFeaturesWpfResources = Microsoft.CodeAnalysis.EditorFeaturesResources;
```

Comment:
added temporarily so that we don't have to touch code files when moving them from EditorFeatures.Wpf down to EditorFeatures.  Will remove in followup pr.